FirebaseBroad/FirebaseBroad.swift

```diff
@@ -5,19 +5,30 @@
 //
 
 import ButterBroad
-import Firebase
+import Foundation
 
 final public class FirebaseBroad: ButterBroad.Analytics {
 
-    public var activationHandler: (() -> Void)? = {
-        FirebaseApp.configure()
+    public lazy var activationHandler: (() -> Void)? = {
     }
```

Comment:
Why did you remove `FirebaseApp.configure()` call? It change consistency of broads behavior. Is it forced by Firebase behavior?

Reply:
Current solution implies that FirebaseBroad has not direct references to firebase. More over, some kind of developers want to call `FirebaseApp.configure()` from their own code